roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -65,6 +65,8 @@ spec:
             # Set Felix endpoint to host default action to ACCEPT.
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
               value: "{{ calico_endpoint_to_host_action|default('RETURN') }}"
+            - name: FELIX_HEALTHHOST
+              value: "{{ calico_healthhost|default('localhost') }}"
```

Comment:
`calico_healthhost` should be set in the defaults file along with the other calico settings.

Reply:
fixed